security/crowdsec/Makefile

```
@@ -0,0 +1,9 @@
+PLUGIN_NAME=		crowdsec
+PLUGIN_VERSION=		1.0.0
+#PLUGIN_REVISION=	1
+PLUGIN_DEPENDS=		crowdsec crowdsec-firewall-bouncer
```

```suggestion
PLUGIN_DEPENDS=		crowdsec
```

I think it's already implied

---

Oh yes, now it is, thanks